cmd/kube-bench-summarizer/summarizer/summarizer.go

```diff
@@ -0,0 +1,200 @@
+package summarizer
+
+import (
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+
+	"encoding/json"
+	"fmt"
+	kb "github.com/aquasecurity/kube-bench/check"
+	"os/exec"
+	"path/filepath"
+)
+
+const (
+	DEFAULT_OUTPUT_FILE_NAME = "report.json"
```

Comment:
can you think of a use case where we want to parameterize this param? Given its name starts with DEFAULT, assuming that we consider it could be changed.

Reply:
The file name is being passed here: https://github.com/rancher/cis-k8s/pull/1/files#diff-95f2857257f0727f64e76eadc2945514R33 ... in case we change it in future or use some other tool.